src/ShadowDOM/wrappers/Range.js

```diff
@@ -20,23 +20,69 @@
 
   var OriginalRange = window.Range;
 
+  var ShadowRoot = scope.wrappers.ShadowRoot;
+
+  function isElement(node){
```

Comment:
found this post from our very own @addyosmani a few days ago:
https://medium.com/@addyosmani/auto-formatting-javascript-code-style-fe0f98a923b8

I haven't tried it on this code base but I'd be happy to accept auto reformatting with that tool (I can try too). Yeah, the code here might not be perfectly consistent, since webcomponentsjs is a few code bases merged together. Maybe we can reformat it all to be consistent. Or at least ShadowDOM codebase